patterns/roks/main.tf

```diff
@@ -15,6 +15,17 @@ provider "ibm" {
 # Landing Zone
 ##############################################################################
 
+data "ibm_is_ssh_keys" "existing_keys" {}
+
+locals {
+  # compare the remote keys with input variable
+  existing_ssh_key_id = { for key in data.ibm_is_ssh_keys.existing_keys.keys : key.name => key.id if key.public_key == replace(var.ssh_public_key == null ? "" : var.ssh_public_key, "/==.*$/", "==") }
```

Comment:
It'd be nice to add a comment on the need for the replace here.

Reply:
Sure. I have added the comment.